README.md

```diff
@@ -1 +1,197 @@
+<<<<<<< HEAD
```

Comment:
- Let's remove these lines added that indicate conflicts in your file.

Reply:
Resolved